custom/icds_reports/management/commands/fix_past_data.py

```diff
@@ -0,0 +1,74 @@
+import os
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
+
+
+from django.core.management.base import BaseCommand
+from corehq.util.argparse_types import date_type
+from django.db import connections, transaction
+from corehq.sql_db.connections import get_icds_ucr_citus_db_alias
+
+
+@transaction.atomic
+def _run_custom_sql_script(command, day=None):
+    db_alias = get_icds_ucr_citus_db_alias()
+    if not db_alias:
+        return
+    with connections[db_alias].cursor() as cursor:
+        cursor.execute(command, [day])
+
+
+class Command(BaseCommand):
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'month',
+            type=date_type,
+            help='The start date (inclusive). format YYYY-MM-DD'
+        )
+
+    def build_data(self, monthly_date):
+        date = monthly_date['record_date']
+        print(f'\n======= Executing for month {date}======\n')
+        for i in range(1, 8):
+            print(f'==============Executing Script {i} =============')
+            if monthly_date["default"] is False and i > 2:
+                path = os.path.join(os.path.dirname(__file__), 'sql_scripts',
+                                'fix_past_data_part_1_{}.sql'.format(i))
```

Comment:
this naming scheme is really confusing, why does part 1_3 come after part 2? it would be easier to understand if instead of a number, that was a descriptive word like "partitioned" so that it was clear what the difference was and for all cases the ordering is clear (1 then 2 then 3 etc) rather than (1 then 2 then 1_3)

Reply:
so naming scheme is simple basically for one set we are running `_1_{}` and for the other `_{}` since the first two queries are same so we are running same for them that's why using the >2 thing.